src/components/common/WalletInfo/index.tsx

```diff
@@ -1,24 +1,20 @@
 import { Box, Typography } from '@mui/material'
-import { Suspense, useMemo } from 'react'
+import { Suspense } from 'react'
 import type { ReactElement } from 'react'
 
 import EthHashInfo from '@/components/common/EthHashInfo'
 import WalletIcon from '@/components/common/WalletIcon'
 import type { ConnectedWallet } from '@/hooks/wallets/useOnboard'
 import { useAppSelector } from '@/store'
 import { selectChainById } from '@/store/chainsSlice'
-import { getShortName } from '@/utils/chains'
 
 import css from './styles.module.css'
 
 export const UNKNOWN_CHAIN_NAME = 'Unknown'
 
 const WalletInfo = ({ wallet }: { wallet: ConnectedWallet }): ReactElement => {
   const walletChain = useAppSelector((state) => selectChainById(state, wallet.chainId))
-
-  const prefix = useMemo(() => {
-    return walletChain?.shortName || getShortName(wallet.chainId)
-  }, [walletChain?.shortName, wallet.chainId])
+  const prefix = walletChain?.shortName
```

Comment:
Here's another instance related to showing prefixes of chains not in the configs.

Reply:
Not sure what you mean. If the walletChain isn't found, it won't show any prefix, and that's fine.

Reply:
Previously we were always showing them if they were present in `chains` but admittedly that doesn't make sense if it is an "unknown" network.